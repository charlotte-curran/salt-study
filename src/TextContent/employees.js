export const employeeList = [
  {
    employee_name: "Koshi Hamedi",
    title: "Director of strategy at Microsoft, Co-founder at </salt>",
    footer: `Founder of Donya Labs and Director of strategy at Microsoft.
                Koshi came to Sweden in the 80's as a refugee. Through his entrepreneurial drive and determination,
                he built one of Sweden's fastest growing technology companies and established the
                company as the global market leader. Donya Labs was acquired by Microsoft in 2017.`,
    img: "koshi_hamedi.png"
  },
  {
    employee_name: "Nicole Henning",
    title: "COO at </salt>",
    footer: `Nicole is a people person with a passion for business and developing others!
                She has worked as a consultant within communication and leadership at several of Sweden's largest
                companies within tech and telecom. Most recently she comes from the recruitment industry.
                Core values for Nicole are teamwork, trust, ethics, and that there's always a reason to celebrate!`,
    img: "nicole_henning.png"
  },

  {
    employee_name: "Marcus Hammarberg",
    title: "Head of curriculum & agile coach at </salt>",
    footer: `In parallel with </salt> Marcus is an agile coach at Spotify. As a developer, Marcus contributes
               to open source frameworks and he has also produced several courses on JavaScript development for PluralSight,
               Marcus has also co-authored the book "Kanban In Action". Marcus has worked at companies such as Tradera,
               Huawei and Spotify`,
    img: "marcus_hammarberg.jpg"
  },
  {
    employee_name: "Erşan Curuklu",
    title: "UX instructor at </salt>",
    footer: `Erşan is a highly experienced requirements analyst and UX Designer with over 15 years of experience
                as a digital designer. Erşan is a teacher and lecturere in the UX area, for e.g. at Bergh's School of Communications
                in Stockholm. Erşan has won several awards in UX/UI, including Svenska Design Priset 2014 and Resumé 2014.`,
    img: "ersan_curuklu.png"
  },
  {
    employee_name: "Max Hamedi",
    title: "Professor at KTH & Harvard, Co-founder at </salt",
    footer: `Postdoctorial research fellow at Harvard University & Associate Professor at the Royal Institute of Technology in
                Stockholm. Our in-house doctor and professor scanning for future technologies, driving our efforts to provide the best applicable
                technology training possible.`,
    img: "max_hamedi.png"
  },
  {
    employee_name: "Richard Andemark",
    title: "CEO at </salt>",
    footer: `At age 25 Richard launched a global SaaS company's operations in Finland as Managing Director, reaching profitability
                in just six months. At 29, Richard launched Donya Labs' American operations in San Francisco as GM, achieving exponential
                revenue growth and the company was acquired by Microsoft.`,
    img: "richard_andemark.jpg"
  },
  {
    employee_name: "Daniel Näsman",
    title: "Head instructor at </salt>",
    footer: `Daniel most recently comes from Daniel Wellington where he was Global Head of IT Development, overseeing development offices in Sweden, Poland and China. 
                With many years experience from being both a developer and manager in an international setting he knows how important the developer is on todays and 
                tomorrows market. He is passionate about building teams and making people grow. `,
    img: "daniel_nasman.jpg"
  },
  {
    employee_name: "Mies Mandoki Cirillo",
    title: "Instructor at </salt>",
    footer: `Mies graduated from the first class of </salt> developers. He has a huge passion for the digital space and is always eager to
                learn new tech. Before </salt>, he was working with video editing, graphic design and was the main producer of a large ongoing video series.
                Mies is a calm, highly driven individual who firmly believes that coders are the superstars of the future.`,
    img: "mies_mandoki_cirillo.jpg"
  },
  {
    employee_name: "Zachary Eriksson",
    title: "Junior Instructor at </salt>",
    footer: `Zachary graduated from the third class of developers. His passion for technology and computers has always been close to his heart.
                Before </salt> he worked as a consultant within Wordpress development and design. Zachary is an energetic, social and driven individual who
                believes communication is the key to success.`,
    img: "zachary_eriksson.jpg"
  },
  {
    employee_name: "Adam Dolah",
    title: "Junior Instructor at </salt>",
    footer: `A graduate from the fourth class of </salt>, Adam also has a law degree from Uppsala University. Prior to </salt>, he work for a few years as an 
                associate at two of Sweden's top law firms, and also as a law clerk at Lund District Court. Adam is a very ambitious and team-focused individual, 
                and highly enthusiastic about digitalization in all areas of society.`,
    img: "adam_dolah.jpeg"
  },
  {
    employee_name: "Stina Johansson",
    title: "Head of People & Talent at </salt>",
    footer: `Stina has a passion for people and developing new talents, employees and oganisations! Before </salt> Stina worked as a consultant, project
                manager and with human resources. She has a long experience from the recruitment-, bank-, and IT industries. Core values for Stina is curiosity,
                commitment, courage, professionalism and last but not least to have fun along the way.`,
    img: "stina_johansson.jpg"
  },
  {
    employee_name: "Sara Hjortmar",
    title: "Sales & Marketing </salt>",
    footer: `Sara is an energetic and curious person who has a keen interest in startups. Sara holds a bachelor's degree in business administration
    from Uppsala University, where, in addition to her studies, she was active in corporate promotion organizations. Since then, she has worked with
    marketing at Red Bull and most recently at the tech company Favro. When she is not working, she loves to train and is currently performing a Swedish
    classic.`,
    img: "sara_hjortmar.jpg"
  }
];
