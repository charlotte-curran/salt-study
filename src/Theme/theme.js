export const SaltTheme = {
  color: {
    dark: "#222324",
    blue: "#92b3d2",
    white: "rgba(255,255,255, 1)",
    orange: "#ff7a61",
    shadow: "#545353",
    address: "rgba(255, 255, 255, 0.4)",
    navbar: "rgba(34, 35, 36, 0.7)"
  },
  fonts: {
    futuraPt: '"futura-pt", sans-serif',
    raleway: '"raleway", sans-serif'
  }
};
