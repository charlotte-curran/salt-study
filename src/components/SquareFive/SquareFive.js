import React from "react";
import Fade from "react-reveal/Fade";
import backgroundImage from "../../images/salt-midbg.png";
import { Wrapper, BlueText, TextWrapper } from "./SquareFive.styles";

const SquareFive = props => {
  const { content } = props;
  return (
    <Wrapper background={backgroundImage}>
      {content.map((value, i) => {
        return (
          <TextWrapper key={i}>
            <Fade bottom delay={150 * i}>
              <div key={i}>
                {value[0]}
                <BlueText key={i}>{value[1]}</BlueText>
                {value[2] ? value[2] : null}
              </div>
            </Fade>
          </TextWrapper>
        );
      })}
    </Wrapper>
  );
};

export default SquareFive;
