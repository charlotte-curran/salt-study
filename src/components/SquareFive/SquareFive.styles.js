import styled from "styled-components";

export const Wrapper = styled.div`
  width: 100%;
  height: 100vh;
  display: flex;
  justify-content: center;
  align-items: center;
  background: url(${({ background }) => background}),
    ${({ theme }) => theme.color.shadow};
  background-size: cover;
  background-repeat: no-repeat;
  background-position: center center;
  position: relative;
  flex-wrap: wrap;
  padding-top: 20px;
  padding-bottom: 20px;
  box-sizing: border-box;

  @media (min-width: 900px) and (max-width: 999px) {
    padding: 0 21vw;
  }

  @media (min-width: 1000px) {
    padding-left: 15%;
    padding-right: 15%;
    padding-top: 6.5%;
    padding-bottom: 6.5%;
    height: 100%;
  }

  /* iphone 6/7/8 plus*/ /* iphone 5/se */
  @media (width: 414px) and (height: 736px),
    (height: 568px) and (width: 320px) {
    padding: 0 16px;
  }

  /* targeting landscape oriented phones */
  @media only screen and (min-width: 480px) and (max-width: 896px) and (min-height: 320px) and (max-height: 414px) and (orientation: landscape) {
    min-height: 466px;
  }
`;

export const TextWrapper = styled.div`
  display: inline;
  flex: 1;
  flex-wrap: wrap;
  margin-top: 25px;
  margin-bottom: 25px;
  padding: 0 17px;
  text-align: center;
  color: ${({ theme }) => theme.color.white};
  font-family: ${({ theme }) => theme.fonts.futuraPt};
  text-transform: uppercase;
  font-weight: 800;
  font-size: 18px;
  height: 150px;

  /* Apple iPhone XR 1792x828px at 326ppi Media Queries */
  @media only screen and (width: 414px) and (height: 896px) and (-webkit-device-pixel-ratio: 2) {
    flex-basis: 30%;
  }

  /* Apple iPhone XS Max 2688x1242px at 458ppi Media Queries */
  @media only screen and (width: 414px) and (height: 896px) and (-webkit-device-pixel-ratio: 3) {
    flex-basis: 25%;
  }

  /* targeting landscape oriented phones */
  @media only screen and (min-width: 480px) and (max-width: 896px) and (min-height: 320px) and (max-height: 414px) and (orientation: landscape) {
    height: auto;
  }

  @media (min-width: 500px) and (max-width: 899px) {
    flex-basis: 40%;
  }

  @media (max-width: 500px) {
    padding: 0 21px;
  }

  @media (min-width: 1000px) {
    font-size: 20px;
    padding: 0 30px;
    flex-basis: 25%;
  }

  /* iphone 5/se */
  @media (height: 568px) and (width: 320px) {
    padding: 0;
    margin-top: 10px;
    margin-bottom: 10px;
  }

  /* ipad portrait*/
  @media (width: 768px) and (height: 1024px) {
    padding: 0 60px;
    font-size: 20px;
    box-sizing: border-box;
  }
`;

export const BlueText = styled.span`
  color: ${({ theme }) => theme.color.blue};
`;
