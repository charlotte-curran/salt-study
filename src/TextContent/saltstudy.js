import { employeeList } from "./employees";
export const saltStudyText = {
  splashPage: {
    header: ["Welcome to", " </salt> ", "!"],
    main: "Pick your location:"
  },
  stockholm: {
    1: {
      main: [
        "learn",
        "how to",
        "code",
        "and",
        "launch",
        "your",
        "career",
        "in tech"
      ],
      button: "for companies!"
    },
    2: {
      header: ["apply", "study for 3 months", "get hired on the spot"],
      main: "Welcome to School of applied technology </salt>",
      footer:
        "Apply to our accelerated career program to become a Full Stack JavaScript developer and get a job within our hiring network of companies. Apply now, we fill our seats on a continuous basis!",
      button: "apply now"
    },
    3: {
      main:
        "’’In our 12 week immersive on-site bootcamp students go from novice to an autonomous Full Stack JavaScript developer working directly with companies in the Epicenter ecosystem.’’",
      video: "https://www.youtube.com/embed/d33cSVnktyA"
    },
    4: {
      header: "Partners and hiring network",
      main: ["epicenter", "campus"],
      main2: "Stockholm & Amsterdam",
      footer:
        "Our students are a part of the ecosystem in Epicenter, interacting with the most innovative companies and using real life cases to accelerate learning. Epicenter is a digital House of Innovation and in a unique partnership </salt> and Epicenter are able to provide the most extensive and targeted hiring network for our coders."
    },
    5: [
      [
        "javascript is the",
        " most sought after ",
        "coding language over the next 3-5 years"
      ],
      [
        "we teach you how to code and how to work",
        " in a real life agile dev team"
      ],
      ["demanding and", " accelerated ", "training and career program"],
      ["europe needs", " 500 000 new technology professionals ", "by 2020!"],
      [
        "our instructors want you to succeed and will be there",
        " every step of the way"
      ],
      [
        "build real products and prototypes for",
        " real tech companies ",
        "during your training"
      ]
    ],
    6: {
      circles: [
        {
          header: "let us launch your career in tech",
          footer:
            "The European tech industry needs you. We are currently missing 500.000 technology professionals in Europe and that number is growing fast."
        },
        {
          header: "we will get you hired by the hottest tech companies",
          footer:
            "We are working with the world’s fastest growing digital companies, and they are looking to hire - offering a salary of 30.000 SEK / 3000 EUR/month."
        },
        {
          header: "complete our career program and the education is free",
          footer:
            "We are investing in you, knowing that you will be a successful software developer consultant within our hiring network."
        }
      ],
      footer: {
        header: "here's how it works:",
        bullets: [
          "Apply and take a test",
          "Get accepted (yaay!)",
          "We sponsor your 3 month full-time coding education at Epicenter",
          "Upon graduation we get you a consultant position within our hiring network with a 30,000 SEK / 3000 EUR per month salary",
          "After doing a kick ass job for 12 months, you get permanently hired by your new company"
        ]
      }
    },
    7: {
      header: "Social Responsibility",
      main: "diversity in tech",
      footer:
        "</salt> is striving for a richer diversity in the European tech industry. We believe that professional organizations should mirror the population's diversity. With our accelerated career program we aim to increase the number women and new Europeans in tech."
    },
    8: {
      header: "this is </salt>",
      slider: employeeList
    },
    9: {
      text: [
        {
          header:
            "we help companies with training and provisioning of software developers",
          footer:
            "Let’s develop a predicable and stable recruitment pipeline of software developers to your company. Enable your team to grow with new hires that have the technical and interpersonal skills they need to succeed. We help decrease your new hires' time to productivity by providing practical, job-specific training, simulating real world agile dev teams for 3 months prior to joining your company."
        },
        {
          header: "hire",
          footer:
            "Bring in diverse and technical talent to your company directly from </salt>. We find, test and recruit the most driven and tech savvy talents to our Career Program were we train your future talents with our industry leading instructors."
        },
        {
          header: "onboard",
          footer:
            "Prepare your new hires with the industry expertise, company knowledge, and skills needed to thrive at your company. We will train your new staff so they are ready to become productive in your dev teams."
        },
        {
          header: "upskill & re-skill",
          footer:
            "Transform your team members through rigorous, intensive and customized training programs in order to keep up with new technologies and business challenges."
        }
      ],
      button: "read more"
    },
    10: {
      header: "join us.",
      address: [
        "Applied Technology STHLM AB",
        "Mäster Samuelsgatan 36",
        "c/o Epicenter Stockholm"
      ],
      addressLink:
        "https://maps.google.com/?q=1200 Mäster+Samuelsgatan+36,+111+57+Stockholm",
      button: "tell me more",
      contact:
        "For general questions please shoot us an e-mail at hello@study-at-salt.com",
      footer: `Wow! We currently experiencing a very high volume of applications and emails, so unfortunately we are not able to reply to all e-mails in a timely manner. We apologize for the inconvenience and we will try to respond as soon as possible.`
    },
    11: [
      "campus epicenter",
      "career program",
      "how does it work?",
      "diversity in tech",
      "our team"
    ],
    12: {
      header: [
        "Send me detailed",
        "information about the",
        "2020 career program"
      ],
      main:
        "Enter your email below to receive our Career Pack with detailed information about the program and how to apply.",
      main2: [
        "Just a heads up, it can take up to ",
        "two hours",
        " to receive your Career Pack!"
      ]
    }
  },
  oslo: {
    1: {
      main: [
        "learn",
        "how to",
        "code",
        "and",
        "launch",
        "your",
        "career",
        "in tech"
      ],
      button: "for companies!"
    },
    2: {
      header: [
        "study code in Stockholm for 3 months",
        "get hired and work in Oslo",
        ""
      ],
      main: "School of applied technology </salt>",
      footer:
        "Apply to our accelerated career program based in Stockholm to become a Full Stack JavaScript developer and get a job within our hiring network of companies in Oslo. We'll pay for accommodation during the training and the required travel. Apply now, we fill our seats on a continuous basis!",
      button: "apply now"
    },
    3: {
      main:
        "’’In our 12 week immersive on-site bootcamp students go from novice to an autonomous Full Stack JavaScript developer working directly with companies in the Epicenter ecosystem.’’",
      video: "https://www.youtube.com/embed/d33cSVnktyA"
    },
    4: {
      header: "Partners and hiring network",
      main: ["epicenter", "campus"],
      main2: "Stockholm & Amsterdam",
      footer:
        "Our students are a part of the ecosystem in Epicenter, interacting with the most innovative companies and using real life cases to accelerate learning. Epicenter is a digital House of Innovation and in a unique partnership </salt> and Epicenter are able to provide the most extensive and targeted hiring network for our coders."
    },
    5: [
      [
        "javascript is the",
        " most sought after ",
        "coding language over the next 3-5 years"
      ],
      [
        "we teach you how to code and how to work",
        " in a real life agile dev team"
      ],
      ["demanding and", " accelerated ", "training and career program"],
      ["europe needs", " 500 000 new technology professionals ", "by 2020!"],
      [
        "our instructors want you to succeed and will be there",
        " every step of the way"
      ],
      [
        "build real products and prototypes for",
        " real tech companies ",
        "during your training"
      ]
    ],
    6: {
      circles: [
        {
          header: "let us launch your career in tech",
          footer:
            "The European tech industry needs you. We are currently missing 500.000 technology professionals in Europe and that number is growing fast."
        },
        {
          header: "we will get you hired by the hottest tech companies",
          footer:
            "We are working with the world’s fastest growing digital companies, and they are looking to hire - offering a salary of 40.000 NOK/month."
        },
        {
          header: "complete our career program and the education is free",
          footer:
            "We are investing in you, knowing that you will be a successful software developer consultant within our hiring network. We will pay for your accommodation in Stockholm during the training."
        }
      ],
      footer: {
        header: "here's how it works:",
        bullets: [
          "Apply and take a test",
          "Get accepted (yaay!)",
          "We sponsor your 3 month full-time coding education at Epicenter in Stockholm",
          "Upon graduation we'll get you a consultant position in Oslo within our hiring network with a 40,000 per month salary",
          "After doing a kick ass job for 12 months, you get permanently hired by your new company"
        ]
      }
    },
    7: {
      header: "Social Responsibility",
      main: "diversity in tech",
      footer:
        "</salt> is striving for a richer diversity in the European tech industry. We believe that professional organizations should mirror the population's diversity. With our accelerated career program we aim to increase the number women and new Europeans in tech."
    },
    8: {
      header: "this is </salt>",
      slider: employeeList
    },
    9: {
      text: [
        {
          header:
            "we help companies with training and provisioning of software developers",
          footer:
            "Let’s develop a predicable and stable recruitment pipeline of software developers to your company. Enable your team to grow with new hires that have the technical and interpersonal skills they need to succeed. We help decrease your new hires' time to productivity by providing practical, job-specific training, simulating real world agile dev teams for 3 months prior to joining your company."
        },
        {
          header: "hire",
          footer:
            "Bring in diverse and technical talent to your company directly from </salt>. We find, test and recruit the most driven and tech savvy talents to our Career Program were we train your future talents with our industry leading instructors."
        },
        {
          header: "onboard",
          footer:
            "Prepare your new hires with the industry expertise, company knowledge, and skills needed to thrive at your company. We will train your new staff so they are ready to become productive in your dev teams."
        },
        {
          header: "upskill & re-skill",
          footer:
            "Transform your team members through rigorous, intensive and customized training programs in order to keep up with new technologies and business challenges."
        }
      ],
      button: "read more"
    },
    10: {
      header: "join us.",
      address: [
        "Applied Technology STHLM AB",
        "Mäster Samuelsgatan 36",
        "c/o Epicenter Stockholm"
      ],
      addressLink:
        "https://maps.google.com/?q=1200 Mäster+Samuelsgatan+36,+111+57+Stockholm",
      button: "tell me more",
      contact:
        "For general questions please shoot us an e-mail at hello@study-at-salt.com",
      footer: `Wow! We currently experiencing a very high volume of applications and emails, so unfortunately we are not able to reply to all e-mails in a timely manner. We apologize for the inconvenience and we will try to respond as soon as possible.`
    },
    11: [
      "campus epicenter",
      "career program",
      "how does it work?",
      "diversity in tech",
      "our team"
    ],
    12: {
      header: [
        "Send me detailed",
        "information about the",
        "2020 career program"
      ],
      main:
        "Enter your email below to receive our Career Pack with detailed information about the program and how to apply.",
      main2: [
        "Just a heads up, it can take up to ",
        "two hours",
        " to receive your Career Pack!"
      ]
    }
  }
};
