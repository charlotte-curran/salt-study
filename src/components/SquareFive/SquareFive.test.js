import React from "react";
import { create } from "react-test-renderer";
import SquareFive from "./SquareFive";
import { ThemeProvider } from "styled-components";
import { SaltTheme } from "../../Theme/theme";
import { saltStudyText } from "../../TextContent/saltstudy";

describe("SquareFive", () => {
  test("matches the snapshot", () => {
    const squareFive = create(
      <ThemeProvider theme={SaltTheme}>
        <SquareFive content={saltStudyText[5]} />
      </ThemeProvider>
    );
    expect(squareFive.toJSON()).toMatchSnapshot();
  });
});
