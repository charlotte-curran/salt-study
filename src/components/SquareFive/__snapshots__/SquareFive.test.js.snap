// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SquareFive matches the snapshot 1`] = `
<div
  className="sc-bdVaJa hcqQTd"
>
  <div
    className="sc-bwzfXH NQDGu"
  >
    <div
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      javascript is the
      <span
        className="sc-htpNat bhsidi"
      >
         most sought after 
      </span>
      coding language over the next 3-5 years
    </div>
  </div>
  <div
    className="sc-bwzfXH NQDGu"
  >
    <div
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      we teach you how to code and how to work
      <span
        className="sc-htpNat bhsidi"
      >
         in a real life agile dev team
      </span>
    </div>
  </div>
  <div
    className="sc-bwzfXH NQDGu"
  >
    <div
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      demanding and
      <span
        className="sc-htpNat bhsidi"
      >
         accelerated 
      </span>
      training and career program
    </div>
  </div>
  <div
    className="sc-bwzfXH NQDGu"
  >
    <div
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      europe needs
      <span
        className="sc-htpNat bhsidi"
      >
         500 000 new technology professionals 
      </span>
      by 2020!
    </div>
  </div>
  <div
    className="sc-bwzfXH NQDGu"
  >
    <div
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      our instructors want you to succeed and will be there
      <span
        className="sc-htpNat bhsidi"
      >
         every step of the way
      </span>
    </div>
  </div>
  <div
    className="sc-bwzfXH NQDGu"
  >
    <div
      className="react-reveal"
      style={
        Object {
          "opacity": undefined,
        }
      }
    >
      build real products and prototypes for
      <span
        className="sc-htpNat bhsidi"
      >
         real tech companies 
      </span>
      during your training
    </div>
  </div>
</div>
`;
